addons/missileGuidanceCompat/CfgMissileHOT.hpp

```diff
@@ -0,0 +1,38 @@
+class ace_missileguidance { // reference https://github.com/acemod/ACE3/pull/10029
+    enabled = 1;
+
+    pitchRate = 45;          // Minium flap deflection for guidance
+    yawRate = 45;            // Maximum flap deflection for guidance
+
+    canVanillaLock = 0;          // Can this default vanilla lock? Only applicable to non-cadet mode
+
+    showTrail = 1;
+
+    // Guidance type for munitions
+    defaultSeekerType = "SACLOS";
+    seekerTypes[] = { "SACLOS" };
+
+    defaultSeekerLockMode = "LOAL";
+    seekerLockModes[] = { "LOAL", "LOBL" };
+
+    defaultNavigationType = "Line";
+    navigationTypes[] = { "Line" };
+
+    lineGainP = 7;
+    lineGainD = 6;
+
+    initialPitch = 2;
+
+    seekLastTargetPos = 0;      // seek last target position [if seeker loses LOS of target, continue to last known pos]
+    seekerAngle = 30;           // Angle from the shooter's view that can track the missile
+    seekerAccuracy = 1;         // seeker accuracy multiplier
+
+    seekerMinRange = 75;
+    seekerMaxRange = 4000;      // Range from the missile which the seeker can visually search
+
+    offsetFromCrosshair[] = { 0, 0, 0 }; // where the missile wants to stay in relation to the center of the crosshair.
```

Comment:
I don't have a source for this, but the HOT missile flies 0.3 meters above the crosshair. So I think it should be `{ 0, 0, 0.3 }` for perfect translation. also not blocking

Reply:
Is that a optic offset from the beam projector, or is that how the missile sits in the beam? It doesn't seem there's an offset in [this video](https://www.youtube.com/watch?v=tZQaqCNj4mM), but it also looks like the parameters for the HOT-3 I just copy and pasted may not work the way the missile does.

Reply:
> it doesn't seem there's an offset in [this video](https://www.youtube.com/watch?v=D2jIgR7twmc), but it also looks like the parameters for the HOT-3 I just copy and pasted from your PR may not work the way the missile does (or at least the HOT-1).

Looking at that video again, and some other specs on the HOT missile, I think this is a second helicopter watching it go in

Reply:
HOT3 has the offest, not HOT1/HOT2. This is fine